cmd/telegraf/telegraf.go

```diff
@@ -45,6 +47,8 @@ var fOutputFiltersLegacy = flag.String("outputfilter", "",
 	"filter the outputs to enable, separator is :")
 var fConfigDirectoryLegacy = flag.String("configdirectory", "",
 	"directory containing additional *.conf files")
+var fService = flag.String("service", "",
+	"operate on the service")
```

Comment:
I think this should also be in a `if windows {}` conditional. Otherwise we have this flag which will be present, but does nothing on other OSs.